src/components/Reports/ReportsPageRanking/ReportsPageRanking.styles.js

```diff
@@ -1,7 +1,47 @@
-import React from 'react';
+import styled from 'styled-components';
 
-export const PageRankingItem = ({children, index}) => <div key={index} className="page-ranking--item">{children}</div>;
+export const PageRankingItem = styled.div`
+  border-top: 1px solid #ccc;
+  border-bottom: 1px solid #ccc;
+  background: #f2f2f2;
+  margin-bottom: 5px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
 
-export const PageRankingItemText = ({children}) => <p>{children}</p>;
+  & > div {
+    padding: 20px 30px;
+  }
 
-export const PageRankingBreakdown = ({children}) => <div>{children}</div>;
\ No newline at end of file
+  a {
+    font-size: 13px;
+  }
+`;
+
+export const PageRankingItemText = styled.p`
+  margin: 5px;
```

Comment:
If I am not wrong, this change also affects real styles, it is not only a refactoring.

Reply:
ok, i change the commit name